app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,213 @@
+        app:srcCompat="@drawable/instagram" />
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flow_center"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:constraint_referenced_ids="icon_profile, flow_name_career, text_home_address"
+        app:flow_verticalBias="0.4"
+        app:flow_verticalGap="@dimen/margin_smallest"
+        app:flow_verticalStyle="packed"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="@id/top_view"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"/>
+
+    <de.hdodenhof.circleimageview.CircleImageView
+        android:id="@+id/icon_profile"
+        android:layout_width="0dp"
+        android:layout_height="0dp"
+        android:src="@drawable/icon1"
+        app:layout_constraintBottom_toBottomOf="@id/top_view"
+        app:layout_constraintDimensionRatio="1:1"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toTopOf="parent"
+        app:layout_constraintWidth_percent="0.1"/>
+
+    <androidx.constraintlayout.helper.widget.Flow
+        android:id="@+id/flow_name_career"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="vertical"
+        app:constraint_referenced_ids="text_first_name_sure_name, text_career"
+        app:flow_verticalGap="@dimen/margin_smallest"
+        app:flow_verticalStyle="spread_inside"
+        app:layout_constraintEnd_toEndOf="@id/guideline_end"
+        app:layout_constraintStart_toStartOf="@id/guideline_start"
+        app:layout_constraintTop_toBottomOf="@id/icon_profile"/>
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/text_first_name_sure_name"
+        style="@style/TextAppearance.AppCompat.White18"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/first_name_sure_name"
+        app:layout_constraintEnd_toEndOf="@id/flow_name_career"
+        app:layout_constraintStart_toStartOf="@id/flow_name_career"
+        app:layout_constraintTop_toTopOf="@id/flow_name_career"/>
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/text_career"
+        style="@style/TextAppearance.AppCompat.LogOut14"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:text="@string/career"
+        app:layout_constraintEnd_toEndOf="@id/flow_name_career"
+        app:layout_constraintStart_toStartOf="@id/flow_name_career"
+        app:layout_constraintBottom_toBottomOf="@id/flow_name_career"/>
+
+    <androidx.appcompat.widget.AppCompatTextView
+        android:id="@+id/text_home_address"
```

Comment:
Love the consistent naming here, though not a big fan of snake case  (not an issue, it's my personal preference 😅)